src/Translator.php

```diff
@@ -63,7 +64,7 @@ public function exists($key)
      */
     public function plural($key, $count, array $vars = [])
     {
-        if ($count == 1) {
+        if ($count === 0 || $count === 1) {
```

Comment:
Not sure if you can just change the comparisons here, though.
See `@param string $count The number of items in the translation.` above,